templates/monitoring/dashboard.tf.tpl

```diff
@@ -373,6 +373,7 @@ module "temporaldashboard_otel" {
   source      = "git@github.com:getoutreach/monitoring-terraform.git//modules/dd-dashboards"
   name        = "Terraform: {{ .Config.Name | title }} Temporal OTEL"
   description = "Managed by terraform in github.com/getoutreach/{{ .Config.Name }}"
+  tags        = ["team:{{ stencilArg "reportingTeam" }}"]
```

Comment:
suggestion: arg syntax (I only know this because I was changing Stencil templates yesterday)
```suggestion
  tags        = ["team:{{ stencil.Arg "reportingTeam" }}"]
```

Reply:
Thank you on catching the syntax error!